2024-01-28 12:00:57,487 INFO    MainThread:970006 [internal.py:wandb_internal():90] W&B internal server running at pid: 970006, started at: 2024-01-28 12:00:57.487767
2024-01-28 12:00:57,489 INFO    WriterThread:970006 [datastore.py:open_for_write():75] open: /home/lijunlin/lijunlin/ESCONV/wandb/run-20240128_120057-190ynnnj/run-190ynnnj.wandb
2024-01-28 12:00:57,490 DEBUG   SenderThread:970006 [sender.py:send():236] send: header
2024-01-28 12:00:57,490 DEBUG   SenderThread:970006 [sender.py:send():236] send: run
2024-01-28 12:00:57,623 WARNING MainThread:970006 [internal.py:wandb_internal():153] Internal process interrupt: 1
2024-01-28 12:00:57,811 WARNING MainThread:970006 [internal.py:wandb_internal():153] Internal process interrupt: 2
2024-01-28 12:00:57,812 ERROR   MainThread:970006 [internal.py:wandb_internal():156] Internal process interrupted.
2024-01-28 12:00:58,001 INFO    SenderThread:970006 [dir_watcher.py:__init__():216] watching files in: /home/lijunlin/lijunlin/ESCONV/wandb/run-20240128_120057-190ynnnj/files
2024-01-28 12:00:58,001 INFO    SenderThread:970006 [sender.py:_start_run_threads():815] run started: 190ynnnj with start time 1706443257
2024-01-28 12:00:58,002 INFO    SenderThread:970006 [sender.py:finish():1114] shutting down sender
2024-01-28 12:00:58,002 INFO    SenderThread:970006 [dir_watcher.py:finish():353] shutting down directory watcher
2024-01-28 12:00:58,489 INFO    HandlerThread:970006 [handler.py:finish():806] shutting down handler
2024-01-28 12:00:58,489 INFO    WriterThread:970006 [datastore.py:close():279] close: /home/lijunlin/lijunlin/ESCONV/wandb/run-20240128_120057-190ynnnj/run-190ynnnj.wandb
2024-01-28 12:00:58,490 INFO    MainThread:970006 [internal.py:handle_exit():80] Internal process exited
